src/allmydata/util/log.py

```diff
@@ -38,28 +39,28 @@ def bytes_to_unicode(ign, obj):
 BAD = log.BAD # 40
 
 
-def msg(*args, **kwargs):
+def msg(*args: Any, **kwargs: Any) -> Union[int, Any]:
     return log.msg(*args, **bytes_to_unicode(True, kwargs))
 
 # If log.err() happens during a unit test, the unit test should fail. We
 # accomplish this by sending it to twisted.log too. When a WEIRD/SCARY/BAD
 # thing happens that is nevertheless handled, use log.msg(failure=f,
 # level=WEIRD) instead.
 
-def err(failure=None, _why=None, **kwargs):
+def err(failure: Optional[Any]=None, _why: Optional[Any]=None, **kwargs: Any) -> Union[int, Any]:
```

Comment:
Is that maybe a [Failure](https://docs.twistedmatrix.com/en/stable/api/twisted.python.failure.Failure.html)?

Reply:
Or is the [legacy logging system](https://docs.twistedmatrix.com/en/stable/core/howto/logging.html) all untyped?